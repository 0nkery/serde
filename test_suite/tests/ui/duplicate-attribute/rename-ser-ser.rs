#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
struct S {
    #[serde(rename(serialize = "x", serialize = "y"))]
    x: (),
}

fn main() {}
