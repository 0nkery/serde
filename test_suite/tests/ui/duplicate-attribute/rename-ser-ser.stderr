error: duplicate serde attribute `rename`
 --> $DIR/rename-ser-ser.rs:4:10
  |
4 | #[derive(Serialize)]
  |          ^^^^^^^^^

error: aborting due to previous error

